services/QuillCMS/app/controllers/responses_controller.rb

```diff
@@ -185,7 +185,7 @@ def clone_responses
 
   def reindex_responses_updated_today_for_given_question
     question_uid = params[:question_uid]
-    Response.__elasticsearch__.import query: -> { where("question_uid = ? AND updated_at >= ?", question_uid, Time.zone.now.beginning_of_day) }
+    Response.__elasticsearch__.import({query: -> { where("question_uid = ? AND updated_at >= ?", question_uid, Time.zone.now.beginning_of_day) } })
```

Comment:
The addition of `{` and `}` seems [unnecessary](https://makandracards.com/makandra/496481-changes-to-positional-and-keyword-args-in-ruby-3-0#section-what-changed)

Since the tests stub out these methods, can you verified on staging that these are required?